### PR TITLE
IGUK-540 Adding great design system and using in international

### PR DESCRIPTION
**Pull request by stuart-mindt**
## What
Adding great design system and using in international
## Why
To align design between domestic and international. 

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/IGUK-540
- [x] Jira ticket has the correct status.
- [x] A clear/description pull request messaged added.

### Reviewing help

- [x] Explains how to test locally, including how to set up appropriate data
- [ ] Includes screenshot(s) - ideally before and after, but at least after
- [x] Documentation has been updated as necessary
- [x] Where a PR contains code changes developed or maintained by multiple squads a representative from those squads should review the PR.

### Housekeeping

- [x] Added all new environment variables to Vault.
- [x] Cleaned up old feature flags
- [x] Upgraded any vulnerable dependencies.
- [x] I have updated security dependencies
- [x] Python requirements have been re-compiled.
- [x] I have checked that my PR is using the latest package versions of: great-components, directory-constants, directory-healthcheck, directory-validators, directory-components, directory-api-client, directory-ch-client, django-staff-sso-client, directory-forms-api-client, directory-sso-api-client, sigauth

### Security
- [x] Frontend assets have been re-compiled
- [x] Checked for potential security vulnerabilities
- [x] Ensured any sensitive data is handled appropriately

### Performance
- [x] Evaluated the performance impact of the changes
- [x] Ensured that changes do not negatively affect application scalability.

### Merging

- [] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: international_online_offer/templates/eyb/index.html [hunk 1]
```diff
@@ -14,18 +14,17 @@ <h1 class="govuk-heading-xl govuk-!-margin-bottom-6">How to successfully expand
...
+                             width="19"
                              height="19"
-                             viewBox="0 0 33 40"
-                             aria-hidden="true"
-                             focusable="false">
-                            <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
+                             viewBox="0 0 17 17"
+                             fill="currentColor"
+                             class="great-ds-icon--svg">
+                            <path d="M7.15792 2.6001L7.85005 1.90746C8.14311 1.61418 8.617 1.61418 8.90694 1.90746L14.9677 7.9696C15.2608 8.26288 15.2608 8.73712 14.9677 9.02728L8.90694 15.0925C8.61388 15.3858 8.13999 15.3858 7.85005 15.0925L7.15792 14.3999C6.86175 14.1035 6.86798 13.6199 7.17039 13.3297L10.9272 9.748H1.96699C1.55234 9.748 1.21875 9.41416 1.21875 8.9992V8.0008C1.21875 7.58584 1.55234 7.252 1.96699 7.252H10.9272L7.17039 3.67025C6.86486 3.38009 6.85863 2.8965 7.15792 2.6001Z" />
```
Comment (timothyPatterson) [inline international_online_offer/templates/eyb/index.html:27]: Is there an Icon button component in the Design System that could be used here instead?

**Review by timothyPatterson — COMMENTED** [review]
LGTM but as discussed I think @boxadesign or @lewis-coulson-dit should review.

File: international_online_offer/templates/eyb/index.html (hunk 1 above)
Comment (stuart-mindt) [reply international_online_offer/templates/eyb/index.html:27]: Theres a button component that allows for an icon yeah but not moving to it yet. This is just a first pass.

File: international/sass/core-class-overrides.scss [hunk 2]
```diff
@@ -8,11 +8,11 @@
 }
 
 .autocomplete__input {
-    font-family: $govuk-font-family;
+    // font-family: $govuk-font-family;
```
Comment (stuart-mindt) [inline international/sass/core-class-overrides.scss:11]: This is a temp change. Will add Noto in when design say so instead or ensure autocompletes are covered in design system too.

**Pull request merged**